data/fileio/CSVStreamWriter.h

```diff
@@ -0,0 +1,100 @@
+/* -*- c-basic-offset: 4 indent-tabs-mode: nil -*-  vi:set ts=8 sts=4 sw=4: */
+
+/*
+    Sonic Visualiser
+    An audio file viewer and annotation editor.
+    Centre for Digital Music, Queen Mary, University of London.
+    This file copyright 2017 Lucas Thompson.
+    
```

Comment:
It's probably copyright QM I'm afraid. (The files saying copyright me have code either predating SV or developed under a different sort of contract.)